device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsDeviceAuthenticationCBS.java

```diff
@@ -115,7 +115,17 @@ void sendAuthenticationMessages() throws TransportException
                     }
                 }
                 // Codes_SRS_AMQPSDEVICEAUTHENTICATIONCBS_12_009: [The function shall set the delivery tag for the sender.]
-                byte[] deliveryTag = String.valueOf(this.nextTag++).getBytes();
+                byte[] deliveryTag = String.valueOf(this.nextTag).getBytes();
```

Comment:
nit: Refactor and reuse the same logic is used in AmqpsSessionDeviceOperation.java